### ISSUE TITLE
bone terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3727]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased bone mineral content of femur
2. decreased bone mineral content of femur
3. increased volumetric bone mineral density of distal femur
4. decreased volumetric bone mineral density of distal femur 
5. increased volumetric bone mineral density of the midshaft of femur
6. decreased volumetric bone mineral density of the midshaft of femur
7. increased compact volumetric bone mineral density of distal femur
8. decreased compact volumetric bone mineral density of distal femur
9. increased trabecular volumetric bone mineral density of distal femur
10. decreased trabecular volumetric bone mineral density of distal femur
11. increased normalized polar moment of inertia of midshaft femur
12. decreased normalized polar moment of inertia of midshaft femur
13. increased ultimate load of femur
14. decreased ultimate load of femur 
15. increased volumetric bone mineral density of femur
16. decreased volumetric bone mineral density of femur
17. abnormal strength of bone 
18. abnormal strength of femur 

>2. Synonyms (e.g., absent spleen)

1. increased femur bone mineral content
2. decreased femur bone mineral content
3. increased distal femur diaphysis volumetric bone mineral density 
4. decreased distal femur diaphysis volumetric bone mineral density 
5. increased midshaft femur volumetric bone mineral density
6. decreased midshaft femur volumetric bone mineral density
7. increased distal femur cortical volumetric bone mineral density; increased cortical volumetric bone mineral density of distal femur
8. decreased distal femur cortical volumetric bone mineral density; decreased cortical volumetric bone mineral density of distal femur
9. increased distal femur trabecular volumetric bone mineral density 
10. decreased distal femur trabecular volumetric bone mineral density
11. increased midshaft femur polar moment of inertia
12. decreased midshaft femur polar moment of inertia
13. increased femur ultimate load
14. decreased femur ultimate load
15. increased femur volumetric bone mineral density
16. decreased femur volumetric bone mineral density
17. abnormal bone strength
18. abnormal femur strength

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. elevation in the amount (usually in grams/cm) of bone mineral divided by a bone-scanned area in the femur
2. reduction in the amount (usually in grams/cm) of bone mineral divided by a bone-scanned area in the femur
3. elevation in the qualitative measurement value of mineral content of bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
4. reduction in the qualitative measurement value of mineral content of bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
5. elevation in the qualitative measurement value of mineral content of bone in the long bone of the thigh, measured in the middle of the diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
6. reduction in the qualitative measurement value of mineral content of bone in the long bone of the thigh, measured in the middle of the diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
7. elevation in the qualitative measurement value of mineral content of compact bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
8. reduction in the qualitative measurement value of mineral content of compact bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
9. elevation in the qualitative measurement value of mineral content of trabecular bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
10. reduction in the qualitative measurement value of mineral content of trabecular bone in the long bone of the thigh, measured in the distal diaphysis.  This is expressed as the amount of mineral per cubic cm of bone (usually in mgHA/cm^3), with results generated from Quantitative computed tomography (QCT) and other tests.
11. elevation of the resistance to rotational force applied to the femur midshaft. It represents an increase in the inherent rotational stiffness of the bone around that axis to which the torque is applied.
12. reduction of the resistance to rotational force applied to the femur midshaft. It represents a decrease in the inherent rotational stiffness of the bone around that axis to which the torque is applied.
13. elevation of the maximum force a femur can tolerate before fracturing under axial compression.
14. reduction of the maximum force a femur can tolerate before fracturing under axial compression.
15.  increase in the mineral mass per unit volume of bone in the long bone of the thigh
16.  decrease in the mineral mass per unit volume of bone in the long bone of the thigh
17.  a change related to the ability of bone to withstand the application of force without yielding or breaking.
18.  a change related to the ability of a femur to withstand the application of force without yielding or breaking.

PMID:16269906; https://phys.libretexts.org/Bookshelves/Conceptual_Physics/Book%3A_Body_Physics_-_Motion_to_Metabolism_(Davis)/07%3A_Strength_and_Elasticity_of_the_Body/7.01%3A_Strength_of_Human_Bones 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. increased bone mineral content (MP:0010123)
2. decreased bone mineral content (MP:0010124)
3. increased volumetric bone mineral density of femur (new term)
4. decreased volumetric bone mineral density of femur (new term) 
5. increased volumetric bone mineral density of femur (new term)
6. decreased volumetric bone mineral density of femur (new term) 
7. increased volumetric bone mineral density of femur (new term)
8. decreased volumetric bone mineral density of femur (new term) 
9. increased volumetric bone mineral density of femur (new term)
10. decreased volumetric bone mineral density of femur (new term) 
11. abnormal strength of femur (new term)
12. abnormal strength of femur (new term)
13. abnormal strength of femur (new term)
14. abnormal strength of femur (new term)
15. increased volumetric bone mineral density (MP:0013615)
16. decreased volumetric bone mineral density (MP:0013616)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by slaulederkind** [comment]
more parent terms:

17. abnormal bone structure (MP:0003795)
18. abnormal bone strength (new term)

**Comment by slaulederkind** [comment]
Preferred term labels 3. to 6. should be 
3. increased distal femur diaphysis total volumetric bone mineral density
4. decreased distal femur diaphysis total volumetric bone mineral density
5. increased midshaft femur total volumetric bone mineral density
6. decreased midshaft femur total volumetric bone mineral density

Suggested parent terms should be:

3. increased volumetric bone mineral density (MP:0013615)
4. decreased volumetric bone mineral density (MP:0013616)
5. increased volumetric bone mineral density (MP:0013615)
6. decreased volumetric bone mineral density (MP:0013616)
7. increased distal femur diaphysis total volumetric bone mineral density (new term)
8. decreased distal femur diaphysis total volumetric bone mineral density (new term)
9. increased distal femur diaphysis total volumetric bone mineral density (new term)
10. decreased distal femur diaphysis total volumetric bone mineral density (new term)
